cs/mineralpacket.go

```diff
@@ -150,13 +149,20 @@ func (packet *MineralPacket) completeMove(rules *Rules, player *Player, planet *
 		}
 	}
 
+	mineralsRecovered := 1.0
 	if damage.Uncaught > 0 {
 		packet.checkTerraform(rules, player, planet, damage.Uncaught)
 		packet.checkPermaform(rules, player, planet, damage.Uncaught)
+		receiverDriverSpeed := 0
+		if planet.Spec.HasStarbase {
+			receiverDriverSpeed = planet.Spec.SafePacketSpeed
+		}
+		percentCaughtSafely := float64((packet.WarpSpeed * packet.WarpSpeed) / (receiverDriverSpeed * receiverDriverSpeed))
```

Comment:
```suggestion
		var percentCaughtSafely float64
		if planet.Spec.HasStarbase && planet.Spec.SafePacketSpeed > 0 {
			percentCaughtSafely = float64((packet.WarpSpeed * packet.WarpSpeed) / (planet.Spec.SafePacketSpeed * planet.Spec.SafePacketSpeed))
		}
```

this is causing a divide by zero error if the player has no receiver on their starbase.